src/library/dlgtagfetcher.cpp

```diff
@@ -86,12 +88,12 @@ void DlgTagFetcher::init() {
     connect(&m_tagFetcher, &TagFetcher::networkError, this, &DlgTagFetcher::slotNetworkResult);
 
     // Resize columns, this can't be set in the ui file
-    results->setColumnWidth(0, 50);  // Year column
-    results->setColumnWidth(1, 160); // Album column
-    results->setColumnWidth(2, 160); // Album artist column
-    results->setColumnWidth(3, 50);  // Track (numbers) column
-    results->setColumnWidth(4, 160); // Title column
-    results->setColumnWidth(5, 160); // Artist column
+    results->setColumnWidth(0, 160); // Title column
+    results->setColumnWidth(1, 160); // Artist column
+    results->setColumnWidth(2, 160); // Album column
+    results->setColumnWidth(3, 50);  // Year column
+    results->setColumnWidth(4, 50);  // Track (numbers) column
+    results->setColumnWidth(5, 160); // Album artist column
```

Comment:
that is just the initial view.
Adjsuting the columns later on might work with
`results->header()->resizeSections(QHeaderView::ResizeToContents);` somewhere in updateStack() I suppose.

Reply:
Thanks for this hint. I'm sure

> > Strangely enough I did not touch the column alignment. But I agree, this doesn't look good - will change it accordingly
> 
> that can be set reliably in the ui file (unless it's touched in c++ later on)

Seems ok to me in the ui file so I guess it must be altered somewhere else.